### PR TITLE
Remove Travis-CI and dead links.

### PR DESCRIPTION
**Pull request by JoseIgnacioTamayo**
Travis-ci and coverage projects are no longer active. Eventually, GitHub Actions can replace TravisCI.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -388,14 +384,12 @@ limitations under the License.
...
 
 
-[img-pyangbind]: http://rob.sh/img/pyblogo_gh.png
 [img-travis]: https://img.shields.io/travis/robshakir/pyangbind.svg
 [img-codecov]: https://img.shields.io/codecov/c/github/robshakir/pyangbind.svg
 [img-pypi]: https://img.shields.io/pypi/v/pyangbind.svg
 [img-license]: https://img.shields.io/pypi/l/pyangbind.svg
 [img-pyversion]: https://img.shields.io/pypi/pyversions/pyangbind.svg
 
-[pyangbind-docs]: http://pynms.io/pyangbind/
```
Comment (robshakir) [inline README.md:398]: These docs are the same as the ones in `docs/` if we just want to update the link to there.

File: README.md [hunk 2]
```diff
@@ -388,14 +384,12 @@ limitations under the License.
   * The managability team at Juniper Networks.
 
 
-[img-pyangbind]: http://rob.sh/img/pyblogo_gh.png
```
Comment (robshakir) [inline README.md:391]: This image is still available at `https://cdn.rob.sh/img/pyblogo_gh.png` [(link)](https://cdn.rob.sh/img/pyblogo_gh.png).

**Review by robshakir — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (JoseIgnacioTamayo) [reply README.md:391]: Done

File: README.md (hunk 1 above)
Comment (JoseIgnacioTamayo) [reply README.md:398]: Done, link updated to /docs

**Pull request merged**